dali/test/python/operator_2/test_python_function.py

```diff
@@ -315,13 +305,6 @@ def test_python_operator_invalid_function():
     invalid_pipe.run()
 
 
-@raises(TypeError, "do not support multiple input sets")
```

Comment:
So it is supported now, right?

Reply:
Nah, just removed too much tests. Reverted it